examples/fastify-http-server-ts/src/index.ts

```diff
@@ -0,0 +1,37 @@
+import fastifyPlugin from 'fastify-plugin';
+import * as store from '@leapfrogtechnology/async-store';
+import Fastify, { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+
+import { storeParamsPlugin, calculateSum } from './plugin';
+
+const fastifyServer: FastifyInstance = Fastify({ logger: true });
+
+fastifyServer.register(fastifyPlugin(store.initializeFastifyPlugin()));
+fastifyServer.register(fastifyPlugin(storeParamsPlugin));
+
+fastifyServer.register((fastifyInstance, opts, done) => {
+  fastifyInstance.register(fastifyPlugin(calculateSum));
```

Comment:
Wow, we need to do this? I don't seem to get it.

Reply:
Yes, the first register is for registering the fastify routes. Since we want the plugin ` caluclateSum` to run only on the instance of the registered routes of this fastify instance we have to register the plugin to calculate the sum.
We can alternatively define the routes in next file and import from there but since, this example contains only one route. So, I think it would be logical to put it in main fastify-server file